README.md

```diff
@@ -12,7 +12,11 @@ This mod adds a 'Mods' sub-menu to the bottom of the Options menu as its main fe
 
 By default, any installed plugin's [`ConfigFile`](https://docs.bepinex.dev/api/BepInEx.Configuration.ConfigFile.html) data will be used to generate an ingame menu that can modify those configuration options. At this time, only booleans, enums, and explicit value lists are supported; for more complex configuration including arbitrary integers and floats, manual menu construction is needed.
 
-Plugins can add sub-menus to the main menu through the [`Registry`](Registry.cs) class manually, or by implementing one of the [Plugin](Plugin) interfaces that extend from [`IModMenuInterface`](Plugin/IModMenuInterface.cs) for a simplified workflow. Plugins can add the [`ModMenuIgnore`](Plugin/ModMenuIgnore.cs) attribute to opt out of automatic menu creation.
+Plugins can add sub-menus to the main menu through the [`Registry`](Registry.cs) class manually, or by implementing one of the [Plugin](Plugin) interfaces that extend from [`IModMenuInterface`](Plugin/IModMenuInterface.cs) for a simplified workflow.
+
+Plugins can opt out of automatic menu creation by doing one of:
+* Adding the [`[ModMenuIgnore]`](Models/Attributes.cs) attribute to their plugin class
+* Creating an attribute named `ModMenuIgnoreAttribute` and adding it to their plugin class
```

Comment:
It'd be nice to clarify how to do this for people who have not made custom attributes before. Alternatively, source generator /hj

Reply:
> It'd be nice to clarify how to do this for people who have not made custom attributes before.

That's true, I'll add a code block here but if #11 happens then it should probably go somewhere in the "mod menu cookbook" (which probably needs to exist)